mod-pubsub-client/pom.xml

```diff
@@ -208,12 +196,23 @@
         <groupId>org.folio</groupId>
         <artifactId>domain-models-maven-plugin</artifactId>
         <version>${raml-module-builder.version}</version>
```

Comment:
Do we need this plugin here if it is not generating clients and interfaces?

Reply:
Moved the plugin, thank you